pkg/kf/service-bindings/client.go

```diff
@@ -67,6 +71,14 @@ type Client struct {
 func (c *Client) Create(serviceInstanceName, appName string, opts ...CreateOption) (*apiv1beta1.ServiceBinding, error) {
 	cfg := CreateOptionDefaults().Extend(opts).toConfig()
 
+	if serviceInstanceName == "" {
+		return nil, errors.New("no service instance given. Can not create service binding")
```

Comment:
these would work better as single strings when they get appended together in error chains e.g. `"can't create service binding, no service instance given"`

Reply:
done